### PR TITLE
fix reverting to avoid nfs issue

### PR DESCRIPTION
**Pull request by naumvd95**
watch the state
http://jenkins-tpi.bud.mirantis.net:8080/job/9.x.vcenter.vcenter_dvs_bvt/123/console

### CODE REVIEW

File: servers/partner-ci/builders/workstation.sh [hunk 1]
```diff
@@ -94,26 +94,60 @@ print data
...
 
 configure_nfs(){
   set -x
   create_ssh_endpoint
-  source $VENV_PATH/bin/activate
   for esxi_host in $ESXI_HOSTS; do
-
-    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'storages=$(esxcli storage nfs list | grep nfs | cut -d" " -f1); for storage in $storages; do esxcli storage nfs remove -v $storage; done'
-    echo "nfs storages have been successfully removed for $esxi_host"
+    echo "restart services to avoid connection problems>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
```
Comment (rkhozinov) [inline servers/partner-ci/builders/workstation.sh]: remove >>>

File: servers/partner-ci/builders/workstation.sh [hunk 2]
```diff
@@ -94,26 +94,60 @@ print data
...
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD './sbin/services.sh restart'
+    echo "restart scfsbd-watchdog>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD '/etc/init.d/sfcbd-watchdog stop; rm -rf /var/run/sfcb/*;/etc/init.d/sfcbd-watchdog start'
+    echo "esxcli network firewall set --enabled false, to avoid connection problems>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli network firewall set --enabled false'
+    echo "system syslog hostname get>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli system hostname get'
+    echo "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+   ####uncomment below rows if you want to check esxi-information
```
Comment (rkhozinov) [inline servers/partner-ci/builders/workstation.sh]: remove commented lines

**Review by rkhozinov — CHANGES_REQUESTED** [review]

**Review by rkhozinov — CHANGES_REQUESTED** [review]
- Running ESXShell restart
- Running SSH restart
- Running nscd restart
- Running ntpd restart
- SSH login enabled
- Starting ntpd

File: servers/partner-ci/builders/workstation.sh (hunk 2 above)
Comment (naumvd95) [reply servers/partner-ci/builders/workstation.sh]: need to wrap it by checker DEBUG

File: servers/partner-ci/builders/workstation.sh [hunk 3]
```diff
@@ -94,26 +94,63 @@ print data
...
+    echo "restart scfsbd-watchdog>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD '/etc/init.d/sfcbd-watchdog stop; rm -rf /var/run/sfcb/*;/etc/init.d/sfcbd-watchdog start'
+    echo "esxcli network firewall set --enabled false, to avoid connection problems>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli network firewall set --enabled false'
+    echo "system syslog hostname get>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli system hostname get'
+    echo "++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
+    if [[ ${DEBUG} == "true" ]]; then
+      echo "system syslog config get>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+      python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD 'esxcli system syslog config get'
```
Comment (rkhozinov) [inline servers/partner-ci/builders/workstation.sh]: refactor to function

remote_exec (){
host=$1
command=$2
 python2 $SSH_ENDPOINT $host $ESXI_USER $ESXI_PASSWORD $cmd
 echo "$cmd-------------------------------------------------------"
}

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/builders/workstation.sh [hunk 4]
```diff
@@ -94,26 +94,44 @@ print data
...
+  host=$1
+  cmd=$2
+  echo "$cmd-------------------------------------------------------"
+  python2 $SSH_ENDPOINT $host $ESXI_USER $ESXI_PASSWORD "$cmd"
+
+}
 configure_nfs(){
   set -x
   create_ssh_endpoint
-  source $VENV_PATH/bin/activate
```
Comment (rkhozinov) [inline servers/partner-ci/builders/workstation.sh:104]: revert that

**Review by rkhozinov — CHANGES_REQUESTED** [review]

File: servers/partner-ci/builders/workstation.sh [hunk 5]
```diff
@@ -94,26 +94,44 @@ print data
...
+    if [[ ${DEBUG} == "true" ]]; then
+      esxi_exec $esxi_host 'esxcli storage nfs list'
+      echo "----------------------------------------------------------------------------------------------------"
+    fi
+    esxi_exec $esxi_host 'storages=$(esxcli storage nfs list | grep nfs | cut -d" " -f1); for storage in $storages; do esxcli storage nfs remove -v $storage; done'
 
     for nfs_share in $NFS_SHARES; do
-      python2 $SSH_ENDPOINT $esxi_host $ESXI_USER $ESXI_PASSWORD "esxcli storage nfs add -H $NFS_SERVER -s /var/$nfs_share -v $nfs_share"
-      echo "$nfs_share has been successfully connected for $esxi_host"
+      if esxi_exec $esxi_host "esxcli storage nfs add -H $NFS_SERVER -s /var/$nfs_share -v $nfs_share"; then
```
Comment (rkhozinov) [inline servers/partner-ci/builders/workstation.sh:135]: - do nfs_share addition on esxi host `esxi_exec "for nfs_share in $NFS_SHARES"`...
- create a variable with list of commands and then execute them in cycle
- may be you need to reassign an error-code from paramiko 
- after 'esxcli storage core adapter rescan --all' need to check, that nfs are connected and active 
OR 
you can use a simple ansible-playbook to do all that stuff

**Review by rkhozinov — CHANGES_REQUESTED** [review]

**Review by rkhozinov — APPROVED** [review]

File: servers/partner-ci/builders/workstation.sh (hunk 5 above)
Comment (naumvd95) [reply servers/partner-ci/builders/workstation.sh:135]: 1,2 = seems seems there is no normal code-style cases
3,4=done
3- test http://jenkins-tpi.bud.mirantis.net:8080/job/9.x.vcenter.custom.qa/65/console

**Pull request merged**